### PR TITLE
chore(deps): update deps

### PR DESCRIPTION
**Pull request by krshkun**
Update dependencies, why because:

1) Most of these include bug fixes
2) Some of these are performance improvements (next v13.1)
3) Nothing breaks, but if don't update dependencies regularly then everything will soon.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3108"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]

**Comment by dan-mba** [comment]
I think this PR might not be needed anymore. I did a bunch of updates in my testing PR which got merged today.

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "@headlessui/react": "^1.7.7",
     "@heroicons/react": "^2.0.13",
     "@mdx-js/loader": "^2.2.1",
-    "@mdx-js/react": "^2.2.1",
+    "@mdx-js/react": "^2.1.5",
```
Comment (dan-mba) [inline package.json:10]: This is going to an older version

File: package.json [hunk 2]
```diff
@@ -54,14 +55,13 @@
     ]
   },
   "devDependencies": {
-    "@axe-core/playwright": "^4.5.2",
```
Comment (dan-mba) [inline package.json:57]: This dependency is necessary for Playwright

**Review by dan-mba — CHANGES_REQUESTED** [review]
See 2 change comments

**Comment by dan-mba** [comment]
> @dan-mba I'm kind of still trying to merge the changes because it isn't the easier. I'm marking this PR as draft for right now

I understand, but wanted to make sure this didn't get accidentally merged by someone trying to keep up with the flood of PRs

File: package.json [hunk 3]
```diff
@@ -28,7 +29,8 @@
   },
   "scripts": {
     "dev": "next dev",
-    "dev:docker": "ncp ./.env.docker ./.env && next dev",
+    "predev:docker": "ncp ./.env.docker ./.env",
+    "dev:docker": "next dev",
```
Comment (eddiejaoude) [inline package.json:33]: Not sure what these are for but should be in a separate PR if this PR is for updating deps

**Comment by eddiejaoude** [comment]
Thanks for updating the dependencies 👍 it is good to do these often 💯

**Comment by krshkun** [comment]
I do not know what just happened. I synced the forks

File: package.json (hunk 3 above)
Comment (krshkun) [reply package.json]: Fixed 👍

**Comment by krshkun** [comment]
@dan-mba, now this PR only updates the other two dependencies that haven't been updated 👍

**Comment by eddiejaoude** [comment]
If there are changes in the package file, we need to include changes to the lock file as well in this PR

**Comment by krshkun** [comment]
Yes @eddiejaoude, actually I'm waiting for this PR to approved (ready to merge or something) and then I can package-lock.json so it's merged in quick succession and that there aren't any conflicts

**Review by dan-mba — APPROVED** [review]
Looks good to me.
Just add the lock file

**Comment by eddiejaoude** [comment]
Conflict, we proabaly need to merge this soon after it is recreated. Let's try tomorrow 

Btw don't resolve the conflicts, just regenerated the file

**Pull request merged**